src/compile/mark/encode/aria.ts

```diff
@@ -0,0 +1,79 @@
+import {entries, keys} from '../../../util';
+import {getMarkPropOrConfig} from '../../common';
+import {VG_MARK_INDEX} from './../../../vega.schema';
+import {UnitModel} from './../../unit';
+import {wrapCondition} from './conditional';
+import {textRef} from './text';
+import {tooltipData} from './tooltip';
+
+export function aria(model: UnitModel) {
+  const {markDef, config} = model;
+
+  const enableAria = getMarkPropOrConfig('aria', markDef, config);
+
+  // We can ignore other aria properties if ariaHidden is true.
+  if (enableAria === false) {
+    // getMarkGroups sets aria to false already so we don't have to set it in the encode block
+    return {};
+  }
+
+  return {
+    ...(enableAria ? {aria: enableAria} : {}),
+    ...ariaRoleDescription(model),
+    ...description(model)
+  };
+}
+
+function ariaRoleDescription(model: UnitModel) {
+  const {mark, markDef, config} = model;
+
+  if (config.aria === false) {
+    return {};
+  }
+
+  const ariaRoleDesc = getMarkPropOrConfig('ariaRoleDescription', markDef, config);
+
+  if (ariaRoleDesc != null) {
+    return {ariaRoleDescription: {value: ariaRoleDesc}};
+  }
+
+  return mark in VG_MARK_INDEX ? {} : {ariaRoleDescription: {value: mark}};
```

Comment:
bars and points, for example, are not VG marks.  Should they get no `ariaRoleDescription`?

Reply:
Vega marks don't need a default because Vega already has one.